manifests/charts/cloudcore/templates/rbac_cloudcore_feature.yaml

```diff
@@ -10,8 +10,8 @@ rules:
   resources: ["serviceaccounts"]
   verbs: ["list", "watch"]
 - apiGroups: ["policy.kubeedge.io"]
-  resources: ["serviceaccountaccesses"]
-  verbs: ["list", "watch"]
+  resources: ["serviceaccountaccesses", "serviceaccountaccesses/status"]
+  verbs: ["list", "watch", "create", "update"]
```

Comment:
Does delete verbs involved？

Reply:
Not yet discovered.

Reply:
> Does delete verbs involved？

I found a deletion situation and added a delete.
![67081f20d9a6cc84c7fd97256e5da32](https://github.com/kubeedge/kubeedge/assets/110345347/24a6e796-6b96-427b-9ca4-be32fb047b8f)